build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -145,7 +147,14 @@ function(iree_run_module_test)
       else()
         file(RELATIVE_PATH _OUTPUT_FILE_RELATIVE_PATH
           "${CMAKE_CURRENT_BINARY_DIR}" "${_OUTPUT_FILE_ABS_PATH}")
```

Comment:
I guess we can move this into the else branch below.

Reply:
Done.